rerun_py/rerun_sdk/rerun/archetypes/image_encoded_ext.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         self: Any,
         *,
         path: str | pathlib.Path | None = None,
-        contents: datatypes.BlobLike | None = None,
+        contents: bytes | IO[bytes] | datatypes.BlobLike | None = None,
```

Comment:
`BlobLike` should probably contain the `IO[bytes]` alias? This can be done in its fbs definition with the right python attribute